contracts/HubPool.sol

```diff
@@ -603,60 +603,74 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
 
     function executeRootBundle(
         uint256 chainId,
-        uint256[] memory bundleLpFees,
-        int256[] memory netSendAmounts,
-        int256[] memory runningBalances,
+        uint256[] calldata bundleLpFees,
```

Comment:
Use `calldata` for variables that will just be loaded into an in memory `PoolRebalance` struct